core/domain/wipeout_service.py

```diff
@@ -130,7 +130,7 @@ def _hard_delete_explorations_and_collections(pending_deletion_model):
         force_deletion=True)
 
 
-def _generate_activity_to_user_ids_mapping(activity_ids):
+def _generate_activity_to_pseudonymized_ids_mapping(activity_ids):
     """Generate mapping from activity IDs to user IDs.
```

Comment:
Add "pseudonymous" before "user IDs"

Also need to add somewhere that all this represents a single about-to-be-deleted user. The problem with the current descriptions is that they sound like multiple users may be involved (especially since you talk about multiple user IDs several times) -- this makes things a bit confusing so it may be worth being especially careful about the wording of the comments in this function.